### PR TITLE
Bump version to 0.5.13

### PR DESCRIPTION
**Pull request by MikePresman**
This bump is for the server to push out this fix (already on main: https://github.com/gadget-inc/dateilager/commit/30e04c13a118622abd088fc67b161aa5d3a25df5)

After this PR is merged I will add and push the tag upstream as well as build the docker image and push to GCR

### CODE REVIEW

**Review by angelini — APPROVED** [review]
LGTM, let's see if we can hold off until we know a bit more about the headless-service

**Comment by MikePresman** [comment]
Note: Please do not merge until after this change https://github.com/gadget-inc/global-infrastructure/pull/514 getsdeployed first and applied to dateilager-production.

**Pull request merged**
